packages/modal-ui/src/lib/components/DerivationPath.tsx

```diff
@@ -15,12 +19,16 @@ interface DerivationPathProps {
   onError: (message: string) => void;
 }
 
-export interface HardwareWalletAccountState {
-  derivationPath: string;
-  publicKey: string;
+export type HardwareWalletAccountState = HardwareWalletAccount & {
   accountIds: Array<string>;
-  selectedAccountId: string;
-}
+  selected: boolean;
+};
```

Comment:
So accounts for Ledger are no longer grouped by derivationPath. We can simply express them as:
```ts
export interface HardwareWalletAccountState {
  derivationPath: string;
  publicKey: string;
  accountId: string;
}
```
Of course this means we'll see the same `derivationPath` and `publicKey` when multiple linked accounts are found from the indexer which is fine 🙂